packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -301,18 +304,27 @@ const clickBeginningOfElement = async ( driver, element ) => {
 	await action.perform();
 };
 
+// Clicks in the top left of a text-based element outside of the TextInput
+const clickElementOutsideOfTextInput = async ( driver, element ) => {
+	const location = await element.getLocation();
+	const y = isAndroid() ? location.y - blockEdgeToContent : location.y;
+	const x = isAndroid() ? location.x - blockEdgeToContent : location.x;
+
+	const action = new wd.TouchAction( driver ).press( { x, y } ).release();
+	await action.perform();
+};
+
 // Long press to activate context menu.
 const longPressMiddleOfElement = async ( driver, element ) => {
 	const location = await element.getLocation();
 	const size = await element.getSize();
 
-	const action = await new wd.TouchAction( driver );
 	const x = location.x + size.width / 2;
 	const y = location.y + size.height / 2;
-	action.press( { x, y } );
-	// Setting to wait a bit longer because this is failing more frequently on the CI
-	action.wait( 5000 );
-	action.release();
+	const action = new wd.TouchAction( driver )
+		.longPress( { x, y } )
```

Comment:
I changed this from `press` to `longPress` since it was causing random fails on iOS.